validseqparenthesis.py

```diff
@@ -0,0 +1,26 @@
+def isvalid1(string):
+    dict1={'(':')','[':']','{':'}'}
+    stack=[]
+
+    for i in range(0,len(string)):
+        print(stack)
+        j=len(stack)
+        print(string[i])
+        if(string[i]=='(' or string[i]=='[' or string[i]=='{'):
+            stack.append(string[i])
+        elif(string[i]==')' or string[i]==']' or string[i]=='}'):
+            if(stack==[]):
+                return False
+            if((string[i])==(dict1[stack[j-1]])):
+                stack.pop()
+                '''del stack[j-1]'''
+        else:
+            return False
+    if(stack==[]):
```

Comment:
This will give true in case of empty input, Ask the questioner before hand what should be the output in edge cases or state your assumption.